.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: generic
 
+sudo: false
```

Comment:
According to https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2 this should be
```
sudo: required
```
and not `false`.